### PR TITLE
drop validation and Harvesting HLT sequences in FastSim

### PR DESCRIPTION
**Pull request by mtosi**
as agreed w/ PdmV, this PR is meant to drop the validation and Harvesting sequences related to the HLT from the standard workflow in FastSim

I still have to modify the FastSim workflows, though
this is needed in order to have the Harvesting step properly working
in particular, the command "--fast" should be added

### CODE REVIEW

File: Configuration/StandardSequences/python/Harvesting_cff.py [hunk 1]
```diff
@@ -19,8 +19,16 @@
 dqmHarvestingPOGMC = cms.Path( DQMOffline_SecondStep_PrePOGMC )
 
 validationHarvesting = cms.Path(postValidation*hltpostvalidation*postValidation_gen)
+validationHarvestingNoHLT = cms.Path(postValidation*postValidation_gen)
+
+_validationHarvesting_fastsim = validationHarvesting.copy()
+for _entry in [hltpostvalidation]:
+    _validationHarvesting_fastsim.remove(_entry)
+from Configuration.Eras.Modifier_fastSim_cff import fastSim
+fastSim.toReplaceWith(validationHarvesting,_validationHarvesting_fastsim)
```
Comment (makortel) [inline Configuration/StandardSequences/python/Harvesting_cff.py]: Out of curiosity, why not just
```
fastsim.toReplaceWith(validationHarvesting, validationHarvestingNoHLT)
```
?

File: Configuration/StandardSequences/python/Validation_cff.py [hunk 2]
```diff
@@ -31,22 +31,25 @@
...
+                               *globalhitsanalyze
+                               *globalrechitsanalyze
+                               *globalValidation)
+
+validation = cms.Sequence(validationNoHLT
                          *hltvalidation)
 
 _validation_fastsim = validation.copy()
-for _entry in [globaldigisanalyze,globalhitsanalyze,globalrechitsanalyze]:
+for _entry in [globaldigisanalyze,globalhitsanalyze,globalrechitsanalyze,hltvalidation]:
```
Comment (fwyzard) [inline Configuration/StandardSequences/python/Validation_cff.py]: If you do this, there is no need to define a separate `validationNoHLT` sequence above, right ?
Or do you have other use cases in mind ?

File: Configuration/StandardSequences/python/Validation_cff.py [hunk 3]
```diff
@@ -59,20 +62,40 @@
...
-                          +muonIdValDQMSeq
+                          validation_preprodNoHLT
                           +hltvalidation_preprod
                           )
 
+_validation_preprod_fastsim = validation_preprod.copy()
+for _entry in [hltvalidation_preprod]:
+    _validation_preprod_fastsim.remove(_entry)
+from Configuration.Eras.Modifier_fastSim_cff import fastSim
+fastSim.toReplaceWith(validation_preprod,_validation_preprod_fastsim)
```
Comment (fwyzard) [inline Configuration/StandardSequences/python/Validation_cff.py]: same here - if you use the `fastSim` era to modify `validation_preprod`, there is no need to define a separate `validation_preprodNoHLT` above, right ?

File: Configuration/StandardSequences/python/Validation_cff.py [hunk 4]
```diff
@@ -59,20 +62,40 @@
...
+validation_prod = cms.Sequence(
+             validation_prodNoHLT
             +hltvalidation_prod
             )
 
+_validation_prod_fastsim = validation_prodNoHLT.copy()
+for _entry in [hltvalidation_prod]:
+    _validation_prod_fastsim.remove(_entry)
+from Configuration.Eras.Modifier_fastSim_cff import fastSim
+fastSim.toReplaceWith(validation_prod,_validation_prod_fastsim)
```
Comment (fwyzard) [inline Configuration/StandardSequences/python/Validation_cff.py]: Same here.

**Comment by mtosi** [comment]
I had some issues w/ this PR :(
I opened a new one and I'm closing this

**Pull request closed without merging**